### PR TITLE
[TASK] Move Configuration file description in chapter configuration

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 11.5

### CODE REVIEW

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Configuration files
+===================
+
+The configuration files :file:`LocalConfiguration.php` and
+:file:`AdditionalConfiguration.php` are located in the directory
+:ref:`public/typo3conf/ <directory-public-typo3conf>` in Composer-based
+installations. In legacy installations they are located in
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
+
+The most important configurations file is
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:15]:
```suggestion
The most important configuration file is
```

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+:file:`LocalConfiguration.php`. It contains local settings of the
+main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS`], crucial settings
+like database connect credentials are in here. The file is managed by the
+:guilabel:`Admin Tools`.
+
+The file :file:`LocalConfiguration.php` can be enriched by
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:21]: How can a file be enriched by another file? Maybe better: "The configuration can be adjusted by AdditionalConfiguration" or something similar.

Comment (DavidBruchmann) [reply Documentation/Configuration/ConfigurationFiles.rst:21]: The configuration can be overridden by settings in the file AdditionalConfiguration.php. Despite some downsides this is a widely used option as it allows easily different configuration for various environments like production and development without changing any file: the file LocalConfiguration.php is used for production and developers can create their own configuration in AdditionalConfiguration.php.

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+installations. In legacy installations they are located in
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
+
+The most important configuration file is
+:file:`LocalConfiguration.php`. It contains local settings of the
+main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS`], crucial settings
+like database connect credentials are in here. The file is managed by the
+:guilabel:`Admin Tools`.
+
+The file :file:`LocalConfiguration.php` can be enriched by
```
Comment (linawolf) [inline Documentation/Configuration/ConfigurationFiles.rst:21]:
```suggestion
The file :file:`LocalConfiguration.php` can be overridden by settings in the file
```

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+.. _configuration-files:
+
+===================
+Configuration files
+===================
+
+The configuration files :file:`LocalConfiguration.php` and
+:file:`AdditionalConfiguration.php` are located in the directory
+:ref:`public/typo3conf/ <directory-public-typo3conf>` in Composer-based
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:11]: The reference is not available by now. It is one that is defined in a following PR?
Below the typo3conf ref also.

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+installations. In legacy installations they are located in
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
+
+The most important configuration file is
+:file:`LocalConfiguration.php`. It contains local settings of the
+main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS`], crucial settings
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:17]:
```suggestion
main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS']`, crucial settings
```

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+like database connect credentials are in here. The file is managed by the
+:guilabel:`Admin Tools`.
+
+The file :file:`LocalConfiguration.php` can be overridden by settings in the file
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:21]: Maybe better: "The settings in the LocalConfiguration.php file can be overridden in the AdditionalConfiguration.php file."

The current sentence sounds strange: "The file can be overridden by settings in the file ..."

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+:file:`AdditionalConfiguration.php` which is never touched by TYPO3
+internal management tools. Be aware that having settings within
+:file:`AdditionalConfiguration.php` may prevent the system from doing
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:24]:
```suggestion
:file:`AdditionalConfiguration.php` may prevent the system from performing
```

This avoids "doing" used two times in different contextes in the same sentence.

File: Documentation/Configuration/ConfigurationFiles.rst (hunk 4 above)
Comment (linawolf) [reply Documentation/Configuration/ConfigurationFiles.rst:11]: Yes they are in the follow up. Shall we leave them in?

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
+
+The most important configuration file is
+:file:`LocalConfiguration.php`. It contains local settings of the
+main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS']`, crucial settings
+like database connect credentials are in here. The file is managed by the
+:guilabel:`Admin Tools`.
+
+The file :file:`LocalConfiguration.php` can be overridden by settings in the file
+:file:`AdditionalConfiguration.php` which is never touched by TYPO3
```
Comment (linawolf) [inline Documentation/Configuration/ConfigurationFiles.rst:22]:
```suggestion
The settings in the :file:`LocalConfiguration.php`  can be overridden in the 
:file:`AdditionalConfiguration.php` file, which is never touched by TYPO3
```

File: Documentation/Configuration/ConfigurationFiles.rst (hunk 4 above)
Comment (brotkrueml) [reply Documentation/Configuration/ConfigurationFiles.rst:11]: It would be cleaner. But I am fine with both.

Comment (linawolf) [inline Documentation/Configuration/ConfigurationFiles.rst:11]:
```suggestion
:file:`public/typo3conf/` in Composer-based
```

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+===================
+Configuration files
+===================
+
+The configuration files :file:`LocalConfiguration.php` and
+:file:`AdditionalConfiguration.php` are located in the directory
+:ref:`public/typo3conf/ <directory-public-typo3conf>` in Composer-based
+installations. In legacy installations they are located in
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
```
Comment (linawolf) [inline Documentation/Configuration/ConfigurationFiles.rst:13]:
```suggestion
:file:`typo3conf/`.
```

File: Documentation/Configuration/ConfigurationFiles.rst (hunk 4 above)
Comment (linawolf) [reply Documentation/Configuration/ConfigurationFiles.rst]: done

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**